docs/release-checklist.rst

```diff
@@ -79,6 +92,27 @@ Create Branch and Apply Updates
 - Confirm CI runs successfully on all platforms
 
 
+Preparing to Authenticate Release (Setting up GPG)
+``````````````````````````````````````````````````
```

Comment:
I think this should all be in a separate document, or point to an online tutorial or similar? This isn't really a "release" thing per se, it's a hopefully one-time setup that someone can do once (not for every release).

Reply:
Makes sense.